base/applications/mspaint/lang/tr-TR.rc

```diff
@@ -110,7 +111,7 @@ BEGIN
         MENUITEM "K&opyala\tCtrl+C", IDM_EDITCOPY
         MENUITEM "Y&apıştır\tCtrl+V", IDM_EDITPASTE
         MENUITEM SEPARATOR
-        MENUITEM "Crop &selection", IDM_CROPSELECTION
+        MENUITEM "Seçimi &Kırp", IDM_CROPSELECTION
```

Comment:
Change is mandatory:
MENUITEM "Seçimi &Kırp", IDM_CROPSELECTION
would accel collide with
MENUITEM "&Kes\tCtrl+X", IDM_EDITCUT